modules/installation-configuration-parameters.adoc

```diff
@@ -214,6 +214,15 @@ accounts for the dramatically decreased machine performance.
 |The number of control plane machines to provision.
 |A positive integer greater than or equal to `3`. The default value is `3`.
 
+|`credentialsMode`
+|The Cloud Credential Operator (CCO) mode. If no mode is specified, the CCO dynamically tries to determine the capabilities of the provided credentials, with a preference for mint mode on the platforms where multiple modes are supported.
+[NOTE]
+====
+Not all CCO modes are supported for all cloud providers. For more information on
+CCO modes, see the xref:../../operators/operator-reference.adoc#cloud-credential-operator_red-hat-operators[Cloud Credential Operator] page.
```

Comment:
This `xref` cannot be in the module, unfortunately.

I've been dumping mine under an _Additional resources_ header in the assembly, although it isn't ideal.